### PR TITLE
Fix the redirect from mobile to desktop

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
Fix: https://jira.camptocamp.com/browse/GSGMF-759

File: geoportal/c2cgeoportal_geoportal/templates/dynamic.js [hunk 1]
```diff
@@ -67,12 +67,21 @@ wfs_permalink.update(interface_config.get('wfs_permalink', {}))
...
             _query=no_redirect_query
         )
     else:
+        no_redirect_url = request.route_url(
+            interface_config['redirect_interface'],
+            _query=no_redirect_query
+        )
         url = request.route_url(
             interface_config['redirect_interface'],
             _query=no_redirect_query
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/templates/dynamic.js]: What is the difference between `no_redirect_url` and `url` ?
Maybe I miss something, but I do not see it.
As I see, at this point : `no_redirect_url === url`

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/templates/dynamic.js]: no_redirect_url shoudb have the param `no_redirect=t`... => my PR is incorrect :-(

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/templates/dynamic.js]: See  #4492

**Pull request merged**